packages/react/src/components/SideMenu/SideMenuController.tsx

```diff
@@ -29,6 +34,72 @@ export const SideMenuController = (props: {
       useFloatingOptions: {
         open: show,
         placement: "left-start",
+        middleware: [
+          size({
+            apply({ elements }) {
+              // TODO: Need to fetch the block from extension, else it's
+              // always `undefined` for some reason? Shouldn't the `apply`
+              // function get recreated with the updated `block` object each
+              // time it changes?
+              const block =
+                editor.getExtension(SideMenuExtension)?.store.state?.block;
+              if (block === undefined) {
+                return;
+              }
+
+              if (block.type === "heading") {
+                if (!block.props.level || block.props.level === 1) {
+                  elements.floating.style.setProperty("height", "78px");
```

Comment:
we can't base this on pixels here. What if someone changes the font or font-size in css?